docs/getting-started/continuous_integration.md

```diff
@@ -89,6 +89,10 @@ CIFuzz will never report a failure even if it finds a crash in your project.
 This requires the user to manually check the logs for detected bugs. If dry run mode is desired,
 make sure to set the dry-run parameters in both the `Build Fuzzers` and `Run Fuzzers` action step.
 
+`allowed_broken_targets_percentage`: Can be set if you want to set a stricter
```

Comment:
Do we enforce the "stricter" bit?

Reply:
We don't but I say stricter here because I can't really imagine a reason someone would want a less strict value (unless they decide they don't want OSS-Fuzz anymore and only care about CIFuzz, since there OSS-Fuzz builds will break if the higher value actually gets used).

Reply:
I wonder if it should be `allowed-broken-targets-percentage`?